### PR TITLE
Py module

### PR DESCRIPTION
**Pull request by pelikhan**
Fix python for Arcade "SpriteKind" support. Support for
```
@namespace
class SpriteKind:
   Taco = ...
```
compiles down to 
```
namespace SpriteKind {
   export const Taco = 
}
```

### CODE REVIEW

File: pxtpy/converter.ts [hunk 1]
```diff
@@ -1020,13 +1020,16 @@ namespace pxt.py {
...
             let topLev = isTopLevel()
             ctx.currClass = n
-            let nodes = [
-                todoComment("keywords", n.keywords.map(doKeyword)),
-                todoComment("decorators", n.decorator_list.map(expr)),
-                B.mkText(topLev ? "class " : "export class "),
-                quote(n.name)
-            ]
-            if (n.bases.length > 0) {
+            n.isNamespace = n.decorator_list.some(d => d.kind == "Name" && (<Name>d).id == "namespace");
```
Comment (mmoskal) [inline pxtpy/converter.ts]:
```suggestion
            n.isNamespace = n.decorator_list.some(d => getFullName(d) == "namespace");
```

**Comment by mmoskal** [comment]
It should actually be `SpriteKind` not `sprite_kind` (because it's a 'class') so no need to handle that

**Comment by pelikhan** [comment]
Somehow we geneate "sprite_kind.create()"

**Comment by mmoskal** [comment]
to disable snakify of namespaces:
```diff
diff --git a/pxtcompiler/emitter/service.ts b/pxtcompiler/emitter/service.ts
index fc2bb4d7d..cacc06ea1 100644
--- a/pxtcompiler/emitter/service.ts
+++ b/pxtcompiler/emitter/service.ts
@@ -325,7 +325,6 @@ namespace ts.pxtc {
                     r.pyName = snakify(r.name).toUpperCase()
                     break
                 case SymbolKind.Variable:
-                case SymbolKind.Module:
                 case SymbolKind.Method:
                 case SymbolKind.Property:
                 case SymbolKind.Function:
@@ -334,6 +333,7 @@ namespace ts.pxtc {
                 case SymbolKind.Enum:
                 case SymbolKind.Class:
                 case SymbolKind.Interface:
+                case SymbolKind.Module:
                 default:
                     r.pyName = r.name
                     break
```

**Comment by pelikhan** [comment]
@mmoskal ready for review. The getFullName does not support the Name node so i reverted that.

**Pull request merged**